modules/creating-manifest-file-multiple-interfaces-br-ex.adoc

```diff
@@ -0,0 +1,260 @@
+[source,ini]
+----
+[connection]
+id=eno2
+type=ethernet
+interface-name=eno2
+master=bond1
+slave-type=bond
+autoconnect=true
+autoconnect-priority=20
+----
++
+.Example of the second bond interface definition file that is named `bond1.config`
+[source,ini]
+----
+[connection]
+id=bond1
+type=bond
+interface-name=bond1
+autoconnect=true
+connection.autoconnect-slaves=1
+autoconnect-priority=20
+
+[bond]
+mode=802.3ad
+miimon=100
+xmit_hash_policy="layer3+4"
+
+[ipv4]
+method=auto
+----
++
+.. Convert the definition files to Base64 encoded strings by running the following command:
++
+[source,terminal]
+----
+$ base64 <directory_path>/en01.config
+----
++
+[source,terminal]
+----
+$ base64 <directory_path>/eno2.config
+----
++
+[source,terminal]
+----
+$ base64 <directory_path>/bond1.config
+----
+
+. Prepare the environment variables. Replace `<machine_role>` with the node role, such as `worker`, and replace `<interface_name>` with the name of your additional `br-ex` bridge name.
++
+[source,terminal]
+----
+$ export ROLE=<machine_role>
+----
++
+[source,terminal]
+----
+$ export INTERFACE=<interface_name>
+----
+
+. Define each interface definition in a machine configuration manifest file:
++
+.Example of a machine configuration file with definitions added for `bond1`, `eno1`, and `en02`
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: 12-${ROLE}-sec-bridge-cni
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+      - contents:
+          source: data:;base64,<base-64-encoded-contents-for-bond1.conf>
+        path: /etc/NetworkManager/system-connections/bond1.nmconnection
+        filesystem: root
+        mode: 0600
+      - contents:
+          source: data:;base64,<base-64-encoded-contents-for-eno1.conf>
+        path: /etc/NetworkManager/system-connections/eno1.nmconnection
+        filesystem: root
+        mode: 0600
+      - contents:
+          source: data:;base64,<base-64-encoded-contents-for-eno2.conf>
+        path: /etc/NetworkManager/system-connections/eno2.nmconnection
+        filesystem: root
+        mode: 0600
+# ...
+----
+
+. Create a machine configuration manifest file for configuring the network plugin by entering the following command in your terminal:
++
+[source,terminal]
+----
+$ oc create -f <machine_config_file_name>
+----
+
+. Create an Open vSwitch (OVS) bridge, `br-ex1`, on nodes by using the OVN-Kubernetes network plugin to create an `extra_bridge` file`. Ensure that you save the file in the `/etc/ovnk/extra_bridge` path of the host. The file must state the interface name that supports the additional bridge and not the default interface that supports `br-ex`, which holds the primary IP address of the node. 
++
+.Example configuration for the `extra_bridge` file, `/etc/ovnk/extra_bridge`, that references a secondary interface that is attached to a VLAN
```

Comment:
Example configuration for the `extra_bridge` file, `/etc/ovnk/extra_bridge`, that references a secondary interface ~~that is attached to a VLAN~~

(Remove vlan reference since our example is just `bond1`)